open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/handlers/SubjectAreaCategoryHandler.java

```diff
@@ -317,7 +316,12 @@ public SubjectAreaOMASAPIResponse findCategory(String userId,
         SubjectAreaCategoryRESTServices categoryRESTServices = new SubjectAreaCategoryRESTServices();
         categoryRESTServices.setOMRSAPIHelper(this.oMRSAPIHelper);
 
-        response = this.oMRSAPIHelper.findEntitiesByPropertyValue(methodName, userId, "GlossaryCategory", searchCriteria, asOfTime, offset, pageSize, sequencingOrder, sequencingProperty, methodName);
+        if (searchCriteria == null) {
+            response = oMRSAPIHelper.getEntitiesByType(oMRSAPIHelper, methodName, userId, "GlossaryCategory", asOfTime, offset, pageSize);
```

Comment:
This should not use getEntitiesByType, it needs to use findEntitiesByPropertyValue so the sequencing parameters can be honoured.